helm/dex-app/templates/dex-k8s-authenticator/ingress.yaml

```diff
@@ -20,7 +20,7 @@ spec:
   - hosts:
     {{- if .Values.isWorkloadCluster }}
       - login.{{ .Values.baseDomain }}
-    {{- else }}
+    {{- else if .Values.isManagementCluster }}
```

Comment:
I have modified the ingress templates to specifically check for `ManagementCluster` configuration as well.